Makefile

```
@@ -0,0 +1,96 @@
+		awk 'BEGIN {FS = ":.*?## "}; {printf "  %-14s %s\n", $$1, $$2}'
+	@echo ""
+	@echo "Environments (env=<name>):"
+	@$(MAKE) --no-print-directory list-envs
+
+list-envs: ## List available environments
+	@find $(ENVS_DIR) -name '*.tf' -exec dirname {} \; 2>/dev/null | \
+		sort -u | sed 's#^$(ENVS_DIR)/#  #' || true
+	@echo "  aliases: vpn -> do/vpn"
+
+# Ensure env is set and points at a real Terraform root.
+guard-env:
```

checkmake (0.3.2)

[warning] Target body for "guard-env" exceeds allowed length of 5 lines (10).

(maxbodylength)